gammapy/utils/tests/test_nddata.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+import random
+import numpy as np
+from astropy.units import Unit as u
```

Comment:
The official recommendation is
```python
import astropy.units as u
```
so if you do
```python
from astropy.units import Unit as u
```
that is confusing to Astropy coders.

In Gammapy we have often this:
```python
from astropy.units import Quantity
spam = Quantity(42, 'spam')
```
Maybe use that here?